bpf/lib/common.h

```diff
@@ -256,11 +256,16 @@ enum {
 };
 
 #define NOTIFY_COMMON_HDR \
-	__u8		type; \
-	__u8		subtype; \
-	__u16		source; \
+	__u8		type;		\
+	__u8		subtype;	\
+	__u16		source;		\
 	__u32		hash;
```

Comment:
Presumably this is backwards compatible? I didn't run through code to check but add a note in the commit msg might help if you end up doing another spin of this set.